scripts/test-check-packages.sh

```diff
@@ -5,6 +5,9 @@ set -euxo pipefail
 cleanup() {
   r=$?
 
+  # Dump stack logs
+  elastic-package stack dump -v --output build/elastic-stack-dump/check
+
```

Comment:
Is this something you added just for debugging this PR or do you want to keep it?

Reply:
I added this on purpose because during in the tests the stack is booted twice, but we dumped logs once. I needed this "dump" call to verify if AWS secrets are not leaking.